.github/workflows/ci.yml

```diff
@@ -187,54 +186,52 @@ jobs:
       fail-fast: true
       matrix:
         ember-try-scenario: [
-          ember-lts-3.8,
-          ember-lts-3.16,
-          ember-lts-3.24,
-          ember-lts-3.28,
-          ember-release,
-          ember-beta,
-          ember-default-with-jquery,
-          ember-classic,
-          embroider-safe,
-          embroider-optimized
-        ]
+            ember-lts-3.28,
+            ember-lts-4.12,
+            #ember-release,
```

Comment:
First this add-on should be converted to Octane and then follow up with v5.12 upgrade.